### ISSUE TITLE
Runtime Error: File not found

### ISSUE THREAD

**Issue by sohaildhawan** [moses-palmer/virtual-touchpad#3]
Hello Moses,
I thought I would love this module but, unfortunately I couldn't run it, I installed via pip today so I have 0.16 version. I have python 3.4.3 The server starts and there is no error. But as soon as I open the server's address on my phone I get this error and opens a blank page on my phone screen at the following address. I'm using Android 4.4.4, although not necessary.
I'm running Windows 7 64 bit, Not a virtual env. and I had another general question?
That 192.168.....is a private address, so only the devices connected to my network can work with it? Not with 3g/4g?

python -m virtualtouchpad
Starting server http://192.168.1.101:16080/...
======== Running on http://192.168.1.101:16080/ ========
(Press CTRL+C to quit)
ERROR:virtualtouchpad.routes._static:File . does not exist
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\virtualtouchpad\resource.py", line 172, in
 open_stream
    return open(_abs(path), 'rb')
  File "C:\Python34\lib\site-packages\virtualtouchpad\resource.py", line 109, in
 _abs
    raise FileNotFoundError(path)
FileNotFoundError: .

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\virtualtouchpad\routes\_static.py", line 3
6, in static
    stream = resource.open_stream(filepath)
  File "C:\Python34\lib\site-packages\virtualtouchpad\resource.py", line 180, in
 open_stream
    raise FileNotFoundError(path)
FileNotFoundError: .

**Comment by moses-palmer** [comment]
Hello Sohail,

I will look into this issue later today, and will hopefully be able to provide a fix promptly.

Regarding your question about network access: the application tries to locate a LAN address on your computer, and binds only to that interface. The bound address is set here: https://github.com/moses-palmer/virtual-touchpad/blob/master/lib/virtualtouchpad/__main__.py#L170.

The communication is performed over an unencrypted WebSocket, and no authentication is performed, so you might not want to allow devices outside of your local network anyway. To listen unrestricted, change the above line to set address to '0.0.0.0'. You will also need to configure your NAT to map an external port to the port used by the application. Again, I do not recommend doing this; rather, connect your phone to your WiFi.

**Comment by moses-palmer** [comment]
I have made a new release. It fixes the issue with *FileNotFound* error. The error occurred when accessing `http://<server>/`; if you instead access `http://<server>/index.xhtml`, it works. The new release also includes the missing help files in the *PyPi* package. I have lately focused on stand-alone packaging using *PyInstaller*, so I have not noticed these errors.

Fixed in 0.16.1:

- Actually include help files in *PyPi* package.
- Corrected handling of getting the root.
